testdata/scripts/bitwarden.txt

```diff
@@ -45,6 +50,11 @@ case "$*" in
   "collectionIds": [],
   "revisionDate": "2020-10-28T00:21:02.690Z"
 }
+EOF
+   ;;
+"get attachment filename --itemid item-id --raw")
+    cat <<EOF
+hidden-file-value
```

Comment:
A similar change will be needed to `bin/bw.cmd` below in the same file for Windows. Sorry.